docs/cookbook/platform-sh.rst

```diff
@@ -201,8 +201,14 @@ Remember to have it imported in the config:
 
     # app/config/config.yml
     imports:
+        # - { resource: parameters.yml } <- Has to be place before our new file
         - { resource: parameters_platform.php }
 
+.. warning::
+
+    It is important to place newly created file after importing regular parameters.yml file. Otherwise your database connection will not work.
+Also this will be the file where you should set required parameters for your. Theirs value will be fetched from environmental variables.
```

Comment:
1. Indent is missing :)
2. `... set required parameters for your.` for your what? 😄 
3. `Theirs value...` => `Its value...`